core/src/test/java/org/apache/xtable/delta/TestDeltaSync.java

```diff
@@ -57,7 +57,7 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
-
+import org.apache.spark.sql.delta.DeltaConfigs;
```

Comment:
@ForeverAngry Can you remove these changes to this file if they are no longer required?